dt-impact-tracer/src/edu/washington/cs/dt/impact/Main/ResultsParser.java

```diff
@@ -1,17 +1,527 @@
-package edu.washington.cs.dt.impact.Main;
 
+// package edu.washington.cs.dt.impact.Main;
+
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
 import java.util.ArrayList;
 import java.util.Arrays;
 import java.util.List;
+import java.util.Scanner;
+
+class Project {
```

Comment:
This should be in it's own class. You can make a new class for this under src/edu/washington/cs/dt/impact/data